### PR TITLE
Fix gradle-7.6 upgrade

### PR DESCRIPTION
**Pull request by BhumikaSaini**
Signed-off-by: Bhumika Saini <mail@bhumika.co>

### Description
Completes the Gradle upgrade to `7.6`. Steps followed: [Upgrading the Gradle Wrapper](https://docs.gradle.org/current/userguide/gradle_wrapper.html#sec:upgrading_wrapper).

### Issues Resolved
We upgraded Gradle to `7.6` in [#5381](https://github.com/opensearch-project/OpenSearch/pull/5381). However, the following files were not updated:
* [gradlew.bat](https://github.com/opensearch-project/OpenSearch/blob/main/gradlew.bat)
* [gradle-wrapper.jar](https://github.com/opensearch-project/OpenSearch/blob/main/gradle/wrapper/gradle-wrapper.jar)
* [minimumGradleVersion](https://github.com/opensearch-project/OpenSearch/blob/main/buildSrc/src/main/resources/minimumGradleVersion)

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [x] Commits are signed per the DCO using --signoff
- [x] Commit changes are listed out in CHANGELOG.md file (See: [Changelog](../blob/main/CONTRIBUTING.md#changelog))

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by dblock** [comment]
Looks like some CI is complaining above (EOLs and precommit on Windows). Thanks for fixing!

**Comment by dblock** [comment]
@reta was it just an oversight in https://github.com/opensearch-project/OpenSearch/pull/5381, do we need to document something about gradle upgrades?

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -12,6 +12,7 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 distributionUrl=https\://services.gradle.org/distributions/gradle-7.6-all.zip
+networkTimeout=10000
```
Comment (reta) [inline gradle/wrapper/gradle-wrapper.properties:15]: Why this is needed?

**Comment by reta** [comment]
> @reta was it just an oversight in #5381, do we need to document something about gradle upgrades?

Yeah, I think I didn't touch the Gradle wrapper (minimal amount of changes there), but it is good idea to update anyway. The link from the description has it all, thanks @dblock

**Comment by tlfeng** [comment]
@BhumikaSaini Thank you for your contribution! 👍 I'm afraid that I have to close the PR, because it has been replaced by the commit https://github.com/opensearch-project/OpenSearch/commit/7914c0499048793b55aa7d4d2c1156293e7998be of upgrading Gradle version to 8.0. 
Two files that you mentioned [gradlew.bat](https://github.com/opensearch-project/OpenSearch/blob/main/gradlew.bat) and [gradle-wrapper.jar](https://github.com/opensearch-project/OpenSearch/blob/main/gradle/wrapper/gradle-wrapper.jar) are both updated. While [minimumGradleVersion](https://github.com/opensearch-project/OpenSearch/blob/main/buildSrc/src/main/resources/minimumGradleVersion) has not been updated, I think you could open new PR if you'd like to update it.

**Pull request closed without merging**